middlewares/selectAndAuthzInstanceQuestion.sql

```diff
@@ -39,14 +48,16 @@ SELECT
     to_jsonb(iq) || to_jsonb(iqnag) AS instance_question,
     jsonb_build_object(
         'id', iqi.id,
-        'prev_instance_question_id', iqi.prev_instance_question_id,
-        'next_instance_question_id', next_instance_question_id,
-        'question_number', question_number,
+        'prev', iqi.prev_instance_question,
+        'next', iqi.next_instance_question,
```

Comment:
Can we call these variables `next_instance_question` and `prev_instance_question`? When reading the code it's really hard to know what's going on with generic names like `next/prev` (and I hate variable renaming 😄).

Reply:
Fixed in recent commit